rts/Sim/MoveTypes/HoverAirMoveType.cpp

```diff
@@ -410,7 +410,7 @@ void CHoverAirMoveType::UpdateHovering()
 
 	// randomly drift (but not too far from goal-position)
 	wantedSpeed = (randomWind * math::fabs(owner->unitDef->dlHoverFactor) * 0.5f);
-	wantedSpeed += (smoothstep(0.0f, 20.0f * 20.0f, (goalPos - owner->pos)) * (goalPos - owner->pos));
+	wantedSpeed += (goalPos - owner->pos) * 0.1f;
```

Comment:
But it doesn't. `smoothstep` interpolates between 0,0,0 and the magic 200,200,200.
That is a drift into +X+Y.